config/openhpc.yml

```diff
@@ -50,13 +50,6 @@ slurm_compute:
   num_nodes: "{{ slurm_compute_num_nodes }}"
   user: "centos"
 
-slurm_computex:
```

Comment:
This has been removed because it is not used anywhere?

Reply:
Yep

Reply:
I added `slurm_computex` as a test and managed to accidentally commit it.